packages/kotti-ui/source/kotti-field-date/index.ts

```diff
@@ -2,6 +2,7 @@ import { FIELD_META_BASE_SLOTS } from '../kotti-field/meta'
 import { MetaDesignType } from '../types/kotti'
 import { attachMeta, makeInstallable } from '../utilities'
 
+import { KOTTI_FIELD_DATE_SUPPORTS } from './constants'
```

Comment:
new PR: follow the convention for kt-field-date-* constants and call it `KOTTI_FIELD_DATE_SHARED_SUPPORTS`  
because our `KOTTI_FIELD_DATE_PROPS` are specific to the date field only